surch/organization.py

```diff
@@ -84,92 +78,85 @@ def __init__(
     def init_with_config_file(cls, config_file, verbose=False,
                               print_result=False,
                               remove_cloned_dir=False,
-                              organization_flag=True):
+                              is_organization=True):
         conf_vars = utils.read_config_file(
             config_file=config_file,
             print_result=print_result,
             verbose=verbose,
             remove_cloned_dir=remove_cloned_dir,
-            organization_flag=organization_flag)
+            is_organization=is_organization)
         return cls(**conf_vars)
 
-    def get_github_repo_list(
-            self,
-            url_type='clone_url',
-            repository_type='public',
-            repository_per_page=100):
-        """This method get from GitHub the git url list for cloning
-        """
-        lgr.info(
-            'Retrieving list of repositories for the {0}...'.format(
-                self.object_type))
-        auth = (self.git_user, self.git_password) if self.auth else False
-        repository_data = \
-            requests.get(
-                ORG_DETAILS_API_URL.format(
-                    self.item_type, self.organization), auth=auth)
-        if repository_data.status_code == requests.codes.NOT_FOUND:
+    def _get_org_data(self):
+        response = requests.get(GITHUB_API_URL.format(
```

Comment:
following my comment regarding self.creds type. I suggest not to specify auth as a parameter to requests.get in case self.creds=None